lib/p2p/Pool.ts

```diff
@@ -137,6 +133,21 @@ class Pool extends EventEmitter {
     this.connected = true;
   }
 
+  private detectExternalIPAddress = async () => {
```

Comment:
Keep this camel case so make it `detectExternalIpAddress` with lower case "p"

Reply:
Sure